readthedocs/telemetry/collectors.py

```diff
@@ -92,8 +92,27 @@ def collect(self):
             },
         }
         data["doctool"] = self._get_doctool()
+
+        # NOTE: Check if there are files at `_build/html` (the old output directory)
+        # and log these projects so we can communicate with their maintainers
+        #
+        # This temporal and should be removed in the future once we have
+        # decided what to do with this collected data.
+        self._check_using_old_output_directory()
+
         return data
 
+    def _check_using_old_output_directory(self):
+        code, stdout, stderr = self.run(
+            "ls",
```

Comment:
The more standard way of testing for this path would be:

```suggestion
            "test",
            "-d",
```

Reply:
Or if we're worried it's a symlink:

```suggestion
            "test",
            "-x",
```